draft-ietf-quic-qpack.md

```diff
@@ -210,10 +210,18 @@ acknowledged by the decoder.
 
 ### Blocked Dynamic Table Insertions {#blocked-insertion}
 
+The encoder considers a dynamic table entry to be blocking if it has not yet
+received acknowledgement of insertion of the entry, or if it has not yet
+received acknowledgement of all references to the entry.  Note that a dynamic
+table entry that has never been referenced can still be blocking.  Also note
+that a blocking entry cannot be evicted, which is unrelated to a blocked stream,
```

Comment:
Suggest: 
```
A blocking entry cannot be evicted until its insertion has been acknowledged and there are no unacknowledged references to the entry outstanding.
--new paragraph
Note:
: A blocking entry is unrelated to a blocked stream, where a decoder cannot decode a stream as a result of references to entries that are not yet available.  An encoder that uses the dynamic table has to manage the possibility of blocked entries, whereas a decoder can forbid an encoder from making references to unacknowledged entries.
```

Reply:
I have a slight preference to emphasize that "blocking" is a concept from the encoder's point of view, for the sake of clarity, but I do not mind dropping that.

However, let me rephase that as "an entry is considered blocking and cannot be evicted" so that this sentence acts an a definition of blocking.  Otherwise the property "blocking" is not defined, and it is not clear if after the insertion is acknowledged and so are all references, the entry will become a kind of blocking entry that can be evicted, or an entry that is not blocking any longer.